assets/js/formValidation.js

```diff
@@ -0,0 +1,47 @@
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+
+const setError = (element, message) => {
+  const inputControl = element.parentElement;
+  const errorDisplay = inputControl.querySelector('.error');
+
+  errorDisplay.innerText = message;
+  inputControl.classList.add('error');
+  inputControl.classList.remove('success');
+};
+
+const setSuccess = (element) => {
+  const inputControl = element.parentElement;
+  const errorDisplay = inputControl.querySelector('.error');
+
+  errorDisplay.innerText = '';
+  inputControl.classList.add('success');
+  inputControl.classList.remove('error');
+};
+
+const isValidEmail = (email) => {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email).toLowerCase());
+};
+
+const validateInputs = () => {
+  const emailValue = email.value.trim();
+
+  if (emailValue === '') {
+    setError(email, 'Email is required');
+  } else if (email.value !== email.value.toLowerCase()) {
+    setError(email, 'Email should be lowercased');
+  } else if (!isValidEmail(emailValue)) {
+    setError(email, 'Please provide a valid email address. Characters should be lowercased');
+  } else {
+    setSuccess(email);
+  }
+};
+
+if (form) {
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    validateInputs();
+  });
```

Comment:
- [ ] Nice work implementing the validation :clap: :sunglasses: However when filling the form with the correct email the form doesn't submit, kindly check it